editor/src/core/shared/project-file-types.ts

```diff
@@ -589,18 +589,20 @@ export function codeFile(
   )
 }
 
-export function isTextFile(projectFile: ProjectFile | null): projectFile is TextFile {
-  return projectFile != null && projectFile.type === 'TEXT_FILE'
+export function isTextFile(projectFile: ProjectFile): projectFile is TextFile {
```

Comment:
is this just making the type nicer? or was this necessitated somehow?

Reply:
Ah, it was causing grief with the optics composition.